src/rabbit_mgmt_dispatcher.erl

```diff
@@ -110,16 +110,20 @@ dispatcher() ->
      {"/vhosts",                                               rabbit_mgmt_wm_vhosts, []},
      {"/vhosts/:vhost",                                        rabbit_mgmt_wm_vhost, []},
      {"/vhosts/:vhost/permissions",                            rabbit_mgmt_wm_permissions_vhost, []},
+     {"/vhosts/:vhost/topic-permissions",                      rabbit_mgmt_wm_topic_permissions_vhost, []},
```

Comment:
We also use dashes (e.g. `aliveness-test`, `cluster-name`, and also deprecated `all-configuration`), so I used them for `topic-permissions` and also recently for `global-parameters`. I don't mind changing those 2, but we won't be consistent anyway.
`operator_policies` and `vhost_limits` are in 3.7.0, so maybe we can still change them?